bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BKException.java

```diff
@@ -20,20 +20,16 @@
  */
 package org.apache.bookkeeper.client;
 
-import java.lang.Exception;
+import org.apache.bookkeeper.client.api.BookKeeperException;
 
 /**
- * Class the enumerates all the possible error conditions
- *
+ * Class the enumerates all the possible error conditions.
  */
-
 @SuppressWarnings("serial")
-public abstract class BKException extends Exception {
-
-    private int code;
+public abstract class BKException extends BookKeeperException {
```

Comment:
can you add comment -- this is going to be deprecated soon, and you are encouraged to use the new api class.

Reply:
done